website/docs/components/pagination/partials/guidelines/overview.md

```diff
@@ -0,0 +1 @@
+Use Pagination to let users navigate through content broken down into pages. Usually paired with tables, but works with other types of page content.
```

Comment:
Do we want to capitalize "Pagination"? In most other places in this text it's in lowercase.

Reply:
I'll try to rationalize my use of `Pagination` vs `pagination`

Basically, I'm using lowercase when referring to pagination in context e.g. "If your product only has cursor pagination…" and uppercase when referring to the actual component e.g. "Use Pagination..." (the actual component).

I'm not saying it's right or wrong, but we should have a unified approach IMO. @heatherlarsen thoughts?

Reply:
I see. In some cases it seemed a little less clear but I understand your reasoning.